lib/ui/viewer/search/search_suggestions.dart

```diff
@@ -39,35 +39,37 @@ class SearchSuggestionsWidget extends StatelessWidget {
           borderRadius: const BorderRadius.all(Radius.circular(8)),
           child: Container(
             margin: const EdgeInsets.only(top: 6),
-            constraints: BoxConstraints(
-              maxHeight: MediaQuery.of(context).size.height * 0.5,
+            constraints: const BoxConstraints(
+              maxHeight: 324,
```

Comment:
Wouldn't it be better to have a longer search results list in view for iPads/Tablets?

Reply:
As discussed, will keep this for now to deterministically cut the list view